### DISCUSSION TITLE
New devices appearing in MQTT autodiscovery with level  zwave-js: 12.4.4  (zwave-js-ui: 9.9.0.09b107a)

### DISCUSSION THREAD

**Discussion by alkempter** [zwave-js/zwave-js-ui#3629] [General]
Hi,
I just upgraded to zwave-js-ui 9.9.0.09b107a and zwave-js: 12.4.4
Now I get a lot of new devices discovered (I use them in Domoticz 2024.4) which I cannot relate to the real hardware, sensor values or settings.
The hardware are a Eurotronics thermostatic valve Spirit (node 8) and a Aeon Labs Multisensor 6 ZW100 (node 5).
The newly discovered devices are shown as switches.
I enclose the output as it is displayed in Domoticz.

Index  ID      ___________________________________             Name    ____________________________         Type    _______  Subtype ___ Value

186  _  zwavejs2mqtt_0xf388bec1_8-112-0-4  __  nodeID_8 (nodeID_8_config_switch_4)  __   Light/Switch _ Switch _  On 
185  _ zwavejs2mqtt_0xf388bec1_8-112-0-3  __   nodeID_8 (nodeID_8_config_switch_3)  __   Light/Switch _ Switch _  On 
184  _ zwavejs2mqtt_0xf388bec1_8-112-0-1   __     nodeID_8 (nodeID_8_config_switch_1)  __   Light/Switch _ Switch _  Off 
183  _ zwavejs2mqtt_0xf388bec1_5-112-0-110   __   nodeID_5 (nodeID_5_config_switch_110) __  Light/Switch _ Switch  _ Off
182    zwavejs2mqtt_0xf388bec1_5-112-0-100      nodeID_5 (nodeID_5_config_switch_100)      Light/Switch  Switch   Off 
181    zwavejs2mqtt_0xf388bec1_5-112-0-252      nodeID_5 (nodeID_5_config_switch_252)      Light/Switch  Switch   Off 
180    zwavejs2mqtt_0xf388bec1_5-112-0-103-128  nodeID_5 (nodeID_5_config_switch_103_128)  Light/Switch  Switch   Off 
179    zwavejs2mqtt_0xf388bec1_5-112-0-103-64   nodeID_5 (nodeID_5_config_switch_103_64)   Light/Switch  Switch   Off
178    zwavejs2mqtt_0xf388bec1_5-112-0-103-32   nodeID_5 (nodeID_5_config_switch_103_32)   Light/Switch  Switch   Off 
177    zwavejs2mqtt_0xf388bec1_5-112-0-103-16   nodeID_5 (nodeID_5_config_switch_103_16)   Light/Switch  Switch   Off 
176    zwavejs2mqtt_0xf388bec1_5-112-0-103-1    nodeID_5 (nodeID_5_config_switch_103_1)    Light/Switch  Switch   Off 
175    zwavejs2mqtt_0xf388bec1_5-112-0-102-128  nodeID_5 (nodeID_5_config_switch_102_128)  Light/Switch  Switch   Off
174    zwavejs2mqtt_0xf388bec1_5-112-0-102-64   nodeID_5 (nodeID_5_config_switch_102_64)   Light/Switch  Switch   Off 
173    zwavejs2mqtt_0xf388bec1_5-112-0-102-32   nodeID_5 (nodeID_5_config_switch_102_32)   Light/Switch  Switch   Off 
172    zwavejs2mqtt_0xf388bec1_5-112-0-102-16   nodeID_5 (nodeID_5_config_switch_102_16)   Light/Switch  Switch   Off 
...
170    zwavejs2mqtt_0xf388bec1_5-112-0-102-1    nodeID_5 (nodeID_5_config_switch_102_1)    Light/Switch  Switch   Off 
...
168    zwavejs2mqtt_0xf388bec1_5-112-0-101-128  nodeID_5 (nodeID_5_config_switch_101_128)  Light/Switch  Switch   On 
167    zwavejs2mqtt_0xf388bec1_5-112-0-101-64   nodeID_5 (nodeID_5_config_switch_101_64)   Light/Switch  Switch   On 
166    zwavejs2mqtt_0xf388bec1_5-112-0-101-32   nodeID_5 (nodeID_5_config_switch_101_32)   Light/Switch  Switch   On 
165    zwavejs2mqtt_0xf388bec1_5-112-0-101-16   nodeID_5 (nodeID_5_config_switch_101_16)   Light/Switch  Switch   On 
164    zwavejs2mqtt_0xf388bec1_5-112-0-101-1    nodeID_5 (nodeID_5_config_switch_101_1)    Light/Switch  Switch   On 
163    zwavejs2mqtt_0xf388bec1_5-112-0-57-255   nodeID_5 (nodeID_5_config_switch_57_255)   Light/Switch  Switch   On 
162    zwavejs2mqtt_0xf388bec1_5-112-0-48-128   nodeID_5 (nodeID_5_config_switch_48_128)   Light/Switch  Switch   Off 
161    zwavejs2mqtt_0xf388bec1_5-112-0-48-64    nodeID_5 (nodeID_5_config_switch_48_64)    Light/Switch  Switch   Off 
160    zwavejs2mqtt_0xf388bec1_5-112-0-48-32    nodeID_5 (nodeID_5_config_switch_48_32)    Light/Switch  Switch   Off
159    zwavejs2mqtt_0xf388bec1_5-112-0-48-16    nodeID_5 (nodeID_5_config_switch_48_16)    Light/Switch  Switch   Off 
158    zwavejs2mqtt_0xf388bec1_5-112-0-48-8     nodeID_5 (nodeID_5_config_switch_48_8)     Light/Switch  Switch   Off
157    zwavejs2mqtt_0xf388bec1_5-112-0-48-4     nodeID_5 (nodeID_5_config_switch_48_4)     Light/Switch  Switch   Off 
156    zwavejs2mqtt_0xf388bec1_5-112-0-48-2     nodeID_5 (nodeID_5_config_switch_48_2)     Light/Switch  Switch   Off
155    zwavejs2mqtt_0xf388bec1_5-112-0-48-1     nodeID_5 (nodeID_5_config_switch_48_1)     Light/Switch  Switch   Off
154    zwavejs2mqtt_0xf388bec1_5-112-0-46       nodeID_5 (nodeID_5_config_switch_46)       Light/Switch  Switch   Off
...
152    zwavejs2mqtt_0xf388bec1_5-112-0-40 1     nodeID_5 (nodeID_5_config_switch_40)       Light/Switch  Switch   Off
151    zwavejs2mqtt_0xf388bec1_5-112-0-2        nodeID_5 (nodeID_5_config_switch_2)        Light/Switch  Switch   On 

Unfortunately the separating blanks in the list are not shown correctly, but the name of the devices all begin with 
nodeID_8 (nodeID_8.....     or
nodeID_5 (nodeID_5....

My home id: 4085825217
home hex: 0xf388bec1

Is there a documentation about theses switches and values?
Should I just ignore them or even delete them in Domoticz so it does not get cluttered?

Thanks a lot
best regards
Al

**Comment by robertsLando** [comment]
Hi @alkempter, them are configurations values, support for them has been added in latest versions
